tests/specs/watch.ts

```diff
@@ -300,5 +300,59 @@ export default testSuite(async ({ describe }, { tsx }: NodeApis) => {
 				expect(p.stderr).toBe('');
 			}, 10_000);
 		});
+
+		describe('include', ({ test }) => {
+			test('watch files for changes', async () => {
+				const entryFile = 'index.js';
+				const file = 'file';
+				const value = 'test';
+				const append = `${value}append`;
```

Comment:
I see you're trying to couple these together via variables but I think the test will be much more readable if you just hard-code them